tests/forms_tests/tests/test_validators.py

```diff
@@ -8,38 +8,36 @@
 from django.core.exceptions import ValidationError
 
 
-class UserForm(forms.Form):
```

Comment:
Didn't realize the diff would look so confusing, I just indented the form. Will move to another commit.